applications/FluidDynamicsHydraulicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -26,12 +26,26 @@ void AddCustomUtilitiesToPython(pybind11::module& m)
 {
```

Comment:
Please, avoid all the non-necessary changes in the format as these complicate very much differentiating the new additions.